src/test/java/org/broadinstitute/hellbender/tools/exome/allelefraction/AFModelUnitTest.java

```diff
@@ -0,0 +1,246 @@
+                            final double altMinorLk = AFModel.hetLogLikelihood(altMinorParameters, count, AFIndicator.ALT_MINOR);
+                            final double refMinorLk = AFModel.hetLogLikelihood(refMinorParameters, count, AFIndicator.REF_MINOR);
+                            Assert.assertEquals(altMinorLk, refMinorLk, 1e-10);
+                        }
+                    }
+                }
+            }
+        }
+    }
+
+
+
+    @Test
+    public void testHetLogLikelihoodPOutlierDependence() {
+        final AllelicCount count = new AllelicCount(DUMMY, 11, 37);
+        final double f = 0.25;
+        final double mu = 1.0;
+        final double beta = 100;
+        final double pi1 = 0.1;
+        final double pi2 = 0.2;
+        final double pi3 = 0.3;
+
+        final AFParameters parameters1 = new AFParameters(f, pi1, mu, beta);
+        final AFParameters parameters2 = new AFParameters(f, pi2, mu, beta);
+        final AFParameters parameters3 = new AFParameters(f, pi3, mu, beta);
+
+        final double lk1 = AFModel.hetLogLikelihood(parameters1, count, AFIndicator.ALT_MINOR);
+        final double lk2 = AFModel.hetLogLikelihood(parameters2, count, AFIndicator.ALT_MINOR);
+        final double lk3 = AFModel.hetLogLikelihood(parameters3, count, AFIndicator.ALT_MINOR);
+
+        Assert.assertEquals(lk2 - lk1, log(1 - pi2) - log(1 - pi1), EPSILON);
+        Assert.assertEquals(lk3 - lk2, log(1 - pi3) - log(1 - pi2), EPSILON);
+    }
+
+
+    @Test
+    public void testCollapsedHetLogLikelihood() throws Exception {
+
+    }
+
+    //when read counts are very high the posterior on lambda ought to be very tight, so we can check moments analytically
+    @Test
+    public void testBiasPosteriorMoment() throws Exception {
+        final double pi = 0.01;
+
+
+        for (final double f : Arrays.asList(0.1, 0.2, 0.3)) {
+            for (final double trueBias : Arrays.asList(0.8, 0.9, 1.0, 1.1, 1.2)) {
+                final double mu = trueBias;
+                final double beta = 100;  //prior average bias agrees with the MLE
+                final AFParameters parameters = new AFParameters(f, pi, mu, beta);
+                for (final double n : Arrays.asList(1000, 2000, 3000)) {
+                    final int a = (int) (n * f / (f + (1 - f) * trueBias));
+                    final int r = (int) (n - a);
+                    final AllelicCount count = new AllelicCount(DUMMY, r, a);
+                    final double actualMean = AFModel.biasPosteriorMoment(parameters, count, AFIndicator.ALT_MINOR, 1);
+                    final double actualMeanSquare = AFModel.biasPosteriorMoment(parameters, count, AFIndicator.ALT_MINOR, 2);
+                    Assert.assertEquals(actualMean, trueBias, 1e-2);
+                    Assert.assertEquals(actualMeanSquare, trueBias*trueBias, 5e-2);
+                }
+            }
+        }
+    }
+
+    @Test
+    public void testMCMC() {
+        final int numSamples = 1000;
```

Comment:
this test takes a little on the long side to run on my laptop...maybe cut down on the number of samples?